tests/blobstore_common_tests.py

```diff
@@ -89,6 +89,97 @@ def testList(self):
                  )]
         self.assertEqual(len(items), 1)
 
+    def testListV2(self):
+        """
+        Ensure that the ```list``` method returns sane data.
+        """
+        items = [item for item in
+                 self.handle.list_v2(
+                     self.test_fixtures_bucket,
+                     "test_good_source_data/0",
+                 )]
+        self.assertTrue(len(items) > 0)
+        for item in items:
```

Comment:
why not:

`self.assertIn("test_good_source_data/0", items)`

Reply:
Good question :)